summary/views.py

```diff
@@ -365,6 +375,42 @@ def get_scenario_data(data, run_uuid):
                     results['batt_kw'] = 'not evaluated'
                     results['batt_kwh'] = 'not evaluated'
 
+                # CHP Size
+                if chp is not None:
+                    if chp.get('max_kw') or -1 > 0:
```

Comment:
@t-kwasnik what is the purpose of all the `or -1 > 0` in summary/views?

Reply:
@NLaws I guess it could be 0, just needs to catch cases where CHP was turned off or not assessed at all

Reply:
@t-kwasnik I'm not sure what you mean? The `or -1 > 0` is the same as `or False` so I don't see what purpose it serves.

Reply:
if `chp.get('max_kw')` is `None` then `chp.get('max_kw') or -1 > 0` will resolve to False, otherwise whatever number the `max_kw` was set to will be compared against `0`. This avoids an error comparing `None` to `0`

Reply:
Oh now I get it thanks. I didn't realize that you do not need parentheses for such a comparison.  I would have got it immediately if it were `if (chp.get('max_kw') or -1) > 0`. I'll add those parantheses.